contract/Portkey.Contracts.BingoGameContract/BingoGameContract.cs

```diff
@@ -143,7 +188,8 @@ public override BoolValue Bingo(Hash input)
 
             var usefulHash = HashHelper.ConcatAndCompute(randomHash, playerInformation.Seed);
             // var bitArraySum = SumHash(usefulHash);
-            var bitArraySum = (int)Context.ConvertHashToInt64(usefulHash, 0, 256);
+            List<int> dices = GetDices(usefulHash);
```

Comment:
Can use `var` to be consistent.

Reply:
ok, will change it to var dices = GetDices(usefulHash);